distribution/lib/Standard/Table/0.0.0-dev/src/Columns_To_Keep.enso

```diff
@@ -35,4 +35,4 @@ type Columns_To_Keep
        that variant is only meant to be used as the default value.
     default_widget display:Display=..When_Modified -> Widget =
         make_single_choice display=display <|
-            ["In_Any", "In_All", "In_List"].map c-> [c, ".."+c]
+            ["..In_Any", "..In_All", "..In_List"].map c-> [c, ".."+c]
```

Comment:
Feel like the .map c-> [c, ".."+c] was already trying to add the ..?

Reply:
Oops - should both be `c` now. Fixed.

The old version put name and then `..<Name>` as the code value.
Now both are the same.